CONTRIBUTING.md

```diff
@@ -41,13 +41,9 @@ We really like to receive feature requests, as it helps us prioritize our work.
 Contributing to the source code
 ---------------
 
-InfluxDB follows standard Go project structure. This means that all
-your go development are done in `$GOPATH/src`. GOPATH can be any
-directory under which InfluxDB and all its dependencies will be
-cloned. For more details on recommended go project's structure, see
-[How to Write Go Code](http://golang.org/doc/code.html) and
-[Go: Best Practices for Production Environments](http://peter.bourgon.org/go-in-production/), or you can just follow
-the steps below.
+InfluxDB follows standard Go project structure. This means that all your Go development are done in `$GOPATH/src`. GOPATH can be any directory under which InfluxDB and all its dependencies will be cloned. For full details on the project structure, follow along below.
```

Comment:
"all your Go development **are** done in `$GOPATH/src`" => all your Go development **is** done in `$GOPATH/src`